jenkins/scripts/integration_test.sh

```diff
@@ -54,6 +54,10 @@ KUBERNETES_VERSION_UPGRADE_FROM="${KUBERNETES_VERSION_UPGRADE_FROM:-}"
 KUBERNETES_VERSION_UPGRADE_TO="${KUBERNETES_VERSION_UPGRADE_TO:-}"
 KUBECTL_SHA256="${KUBECTL_SHA256:-}"
 
+if [[ "${REPO_NAME}" == "ironic-image" && "${REPO_BRANCH}" == "main" ]]; then
```

Comment:
Is the REPO_BRANCH defined here

Reply:
It should be there. It was defined here: https://github.com/metal3-io/project-infra/blob/main/jenkins/jobs/prow_integration_tests.pipeline#L44